src/scenarios/shared/sod.py

```diff
@@ -67,7 +68,12 @@ def runtests(self, scenarioname, dirs, artifact=None):
         ]
         sod_args += dirs.split(';')
 
-        RunCommand(sod_args, verbose=True).run()
+        sod_command = RunCommand(sod_args, verbose=True)
+        sod_command.run()
+        
+        zero_size_regex = rf"({(scenarioname or 'Empty Scenario Name')}\s*-\s*Count\s*\|\s*0\.000\s*count)" # Checks if the overall count is zero    
```

Comment:
The `(scenarioname or 'Empty Scenario Name')` part is duplicated here and on a previous line, maybe assign this at the start of the function instead

Reply:
Changing to the json parsing removed the duplicate usage.